iroha/src/maintenance.rs

```diff
@@ -44,6 +47,21 @@ pub enum Health {
     Ready,
 }
 
+impl Display for Health {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            Health::Healthy => write!(f, "Healthy"),
+            Health::Ready => write!(f, "Ready"),
+        }
+    }
+}
+
+impl From<Health> for HttpResponse {
+    fn from(health: Health) -> Self {
```

Comment:
Can we be sure the appropriate `Content-Type` header will be set?

Reply:
The output is just plain text, should i change it for json?

Reply:
That not necessary. It is up to us decide. AFAIK most services that watch healthiness just look at status code, not the body. I think it is just good to have `Content-Header` set whatever actual body is.